pkg/codegen/nodejs/gen.go

```diff
@@ -2127,12 +2130,9 @@ func genNPMPackageMetadata(pkg *schema.Package, info NodePackageInfo) string {
 		Homepage:    pkg.Homepage,
 		Repository:  pkg.Repository,
 		License:     pkg.License,
-		// Ideally, this `scripts` section would include an install script that installs the provider, however, doing
-		// so causes problems when we try to restore package dependencies, since we must do an install for that. So
-		// we have another process that adds the install script when generating the package.json that we actually
-		// publish.
```

Comment:
I have built a project linked against a code generated module (using `yarn link`). It works without running the `install` script. `npm link` does run the `install` script. We can mitigate this issue by having the install script exit cleanly if it is passed a version of `${VERSION}`, since we don't update that until the version is known (and the package is published).